lib/jeff/message.ex

```diff
@@ -99,6 +99,7 @@ defmodule Jeff.Message do
   defp do_scs(:reply, 0x76, _), do: 0x12
   defp do_scs(:command, 0x77, _), do: 0x13
   defp do_scs(:reply, 0x78, _), do: 0x14
+  defp do_scs(:command, id, true) when id in [0x60, 0x64, 0x65, 0x66, 0x67], do: 0x15
```

Comment:
I think this was the most critical change to get things to work -- commands with no data payload need to use SCS 0x15.

It would have been nice to calculate this based on the actual payload length, but that was a bigger refactor than I wanted to do, and there are a fixed number of commands with no payload.